crates/lightbeam/wasmtime/src/lib.rs

```diff
@@ -0,0 +1,334 @@
+//! Support for compiling with Lightbeam.
+//!
+//! This crates provides an implementation of [`Compiler`] in the form of
+//! [`Lightbeam`].
+
+use cranelift_codegen::binemit;
+use cranelift_codegen::ir::{self, ExternalName};
+use cranelift_codegen::isa;
+use cranelift_wasm::{
+    DefinedFuncIndex, DefinedGlobalIndex, DefinedMemoryIndex, DefinedTableIndex, FuncIndex,
+    GlobalIndex, MemoryIndex, SignatureIndex, TableIndex,
```

Comment:
It is possible to move some of this imports into "crates/environ/src/data_structures.rs"?

Reply:
👍

Reply:
Nice. Sorry, I also meant other no `use cranelift_codegen::XXX` above -- they can be re-exported in wasmtime_environ too thus removing direct dependency on cranelift crates.

Reply:
I was thinking of doing that yeah but I realized that lightbeam itself depends on `cranelift-codegen`, so I figured it was probably best to just do the same thing.